backend/db.json

```
@@ -10,40 +10,168 @@
         "Clue": "This crustacean excretes one of the strongest natural adhesives in the world in order to stick to rocks, piers, ships, and even animals like whales.",
         "AdditionalHint" : "This crustacean has a shell-like covering made up of chitin and calcium plates, which give them a protective rigid structure. ",
         "CorrectMessage" : "Barnacle colonies growing on the hulls of ships actually create huge environmental and economic impact. Barnacle growth can increase the weight and drag of a ship in the water by as much as 60%, increasing fuel usage by as much as 40%! However, barnacles are not all bad, they are important intertidal species! Fossilized barnacles that used to hitch a ride on the backs of humpback and gray whales have been used to help reconstruct ancient migration patterns of whale populations millions of years ago and help us compare migration patterns of whales today.",
-        "MediaLink" : "https://upload.wikimedia.org/wikipedia/commons/1/10/Barnacles_encrusting_wood_%28Cayo_Costa_Island%2C_Florida%2C_USA%29_2_%2826353425635%29.jpg",
-        "Credit" : ""
+        "MediaLink" : "https://lindsay-wildlife-apps.zee-aws.net/get-image/0.png",
+        "Credit" : "Photo by James St. John. Uploaded August 29, 2009. https://commons.wikimedia.org/wiki/File:Barnacles_encrusting_wood_(Cayo_Costa_Island,_Florida,_USA)_2_(26353425635).jpg"
       },
       {
         "ArtifactName" : ["California Moray Eel", "Moray Eel"],
         "Clue": "This ocean dwelling animal is unlike other fish, it only has one singular continuous fin and does not have gill covers, so to breath it must constantly open and close its mouth.",
         "AdditionalHint" : "This ocean dwelling animal is a long, slim snakelike fish.",
         "CorrectMessage" : "California Moray Eels have a second set of jaws in their throat that extend forward to grab prey with backwards facing teeth that drag the prey back into the throat. This second set of jaws is called pharyngeal jaws. They use this feeding method to mainly eat fish, but also feed on octopus, lobsters and shrimp in the rocky reefs of southern California.",
-        "MediaLink" : "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/California_Moray_Eel_%2832582219365%29.jpg/800px-California_Moray_Eel_%2832582219365%29.jpg",
-        "Credit" : ""
+        "MediaLink" : "https://lindsay-wildlife-apps.zee-aws.net/get-image/1.png",
```

```suggestion
        "MediaLink" : "https://lindsay-wildlife-apps.zee-aws.net/backend/get-image/1.png",
```

I think these should look like this

---

Fixed!